tests/storage/test_storage.rs

```diff
@@ -411,6 +412,23 @@ fn test_txn_store_gc() {
     store.get_ok(b"k", 25, b"v2");
 }
 
+fn test_txn_store_gc_multiple_keys(n: usize) {
+    let store = new_assertion_storage();
+    for i in 0..n {
+        let key = format!("k{}", i);
+        store.put_ok(&key.as_bytes(), b"value", 5, 10);
+    }
+    store.gc_ok(20);
```

Comment:
All methods of `AssertionStore` are assertions, so I suppose just `gc_ok` won't cause ambiguity.